heat/core/manipulations.py

```diff
@@ -811,6 +813,350 @@ def hstack(tup):
     return concatenate(tup, axis=axis)
 
 
+def pad(array, pad_width, mode="constant", values=0):
+    """
+    Pads tensor with a specific value (default=0).
+    (Not all dimensions supported)
+
+
+    Parameters
+    ----------
+    array : ht.DNDarray
+        tensor which is to be padded
+    pad_width: sequence of 2-element-sequences (sequence = tuple or list)
+        Number of values padded to the edges of each axis. ((before_1, after_1),...(before_N, after_N)) unique pad widths for each axis.
+        Shortcuts:
+            - ((before, after),)  or (before, after)
+                --> before and after pad width for each axis.
+            - (pad_width,) or int
+                --> before = after = pad width for all axes.
+
+        Determines how many elements are padded along which dimension.
+        Therefore:
+        - pad last dimension:       (
```

Comment:
See formatting above

Reply:
@Cdebus I've fixed the formatting for @lenablind here because she has quite some complicated mix of possibilities. It would be great if you could have a look.